app/Services/Circles/CirclesService.php

```diff
@@ -107,4 +111,40 @@ public function saveTags(Circle $circle, array $tags)
 
         $circle->tags()->sync($tags_on_db->pluck('id')->all());
     }
+
+    public function sendSubmitedEmail(User $user, Circle $circle)
+    {
+        Mail::to($user)
+        ->send(
+            (new SubmitedMailable(
+                $circle,
+            ))
+                ->replyTo(config('portal.contact_email'), config('portal.admin_name'))
+                ->subject('参加登録を提出しました')
```

Comment:
メールの件名をひと目見ただけで要件が何かわかると良いですね

```suggestion
                ->subject('【参加登録】「(企画名)」の参加登録を提出しました')
```